File: programming/lab2/src/main/java/pokemons/Sewaddle.java
package pokemons;

import moves.status.Rest;
import moves.status.StringShot;
import ru.ifmo.se.pokemon.Pokemon;
import ru.ifmo.se.pokemon.Type;

public class Sewaddle extends Pokemon {
    public Sewaddle() {
        this("Sewwadle", 1);
    }
    public Sewaddle(String name, int level) {
        super(name, level);
        this.setLevel(level);
        this.setType(Type.BUG, Type.GRASS);
        this.setStats(45, 53, 70, 40, 60, 42);
        this.addMove(new Rest());
        this.addMove(new StringShot());
    }
}

File: programming/lab2/app/src/main/kotlin/org/itmo/pokemons/Sewaddle.kt
package org.itmo.pokemons

import org.itmo.moves.status.Rest
import org.itmo.moves.status.StringShot
import ru.ifmo.se.pokemon.Pokemon
import ru.ifmo.se.pokemon.Type

open class Sewaddle(name: String, level: Int): Pokemon(name, level) {
    init {
        setType(Type.BUG, Type.GRASS)
        setMove(Rest(), StringShot())
        setStats(45.0, 53.0, 70.0, 40.0, 60.0, 42.0)
    }
}